generators/addons/templates/gulpfileCI.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const gulp = require('gulp');
+const build = require('@microsoft/sp-build-web');
+build.addSuppression(`Warning - [sass] The local CSS class 'ms-Grid' is not camelCase and will not be type-safe.`);
+
+build.initialize(gulp);
+var buildConfig = build.getConfig();
```

Comment:
@baywet so you can only get the config when gulp is already running? or can the ```build.initialize``` come after the ```build.initialize('gulp')```

Reply:
as far as I know the only way to access the karma task configuration is to build the configuration. I haven't looked at that in a while but last time I did, I looked in details. Do you see any challenge with this approach?

Reply:
No not really just integrate the gulp file now in the gulp file, it was not deployed previously. Just wanted to know if I should add it before or after the build.

Reply:
"after the build" are you referring to the `gulp build` task or to the meta build you're working on to compose the gulp files?

Reply:
After the default ```build.initialize(gulp);```

Reply:
As far as I remember yes, otherwise the default SPFx tasks are missing from the context